### PR TITLE
WELD-1884 - test PBA and PB is not fired for anonymous class.

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

**Comment by weld-tester** [comment]
Triggering build using a merge of 9e917b1d4f2adb580abc4b5571a6ed8694fc88c1 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 545 outcome was FAILURE using a merge of 9e917b1d4f2adb580abc4b5571a6ed8694fc88c1 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/545
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/545

**Comment by weld-tester** [comment]
Build 545 outcome was FAILURE using a merge of 9e917b1d4f2adb580abc4b5571a6ed8694fc88c1 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/545
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/545

**Comment by weld-tester** [comment]
Build 545 outcome was FAILURE using a merge of 9e917b1d4f2adb580abc4b5571a6ed8694fc88c1 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/545
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/545

**Comment by weld-tester** [comment]
Build 545 outcome was FAILURE using a merge of 9e917b1d4f2adb580abc4b5571a6ed8694fc88c1 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/545
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/545

File: tests-arquillian/src/test/java/org/jboss/weld/tests/extensions/lifecycle/weld1884/TestExtension.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import java.util.List;
+
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.spi.Extension;
+import javax.enterprise.inject.spi.ProcessBean;
+import javax.enterprise.inject.spi.ProcessBeanAttributes;
+
+public class TestExtension implements Extension {
+
+    public static final List<Class> pbaFiredvalues = new ArrayList<>();
```
Comment (jharting) [inline tests-arquillian/src/test/java/org/jboss/weld/tests/extensions/lifecycle/weld1884/TestExtension.java]: Please do not use raw references to generic types. Class, ProcessBeanAttributes, ProcessBean are all generic and should be used that way. Use Class<?>, ProcessBeanAttributes<?> or ProcessBean<?> if you need to.

**Comment by weld-tester** [comment]
Build 545 outcome was FAILURE using a merge of 9e917b1d4f2adb580abc4b5571a6ed8694fc88c1 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/545
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/545

**Comment by weld-tester** [comment]
Build 545 outcome was FAILURE using a merge of 9e917b1d4f2adb580abc4b5571a6ed8694fc88c1 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/545
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/545

**Comment by weld-tester** [comment]
Triggering build using a merge of bb12eb9f5dafed8131afaf2833ee93eef849ae8c on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 551 is now running using a merge of bb12eb9f5dafed8131afaf2833ee93eef849ae8c on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/551

**Comment by weld-tester** [comment]
Triggering build using a merge of 031f15c2aff68b9c6fbf0a25412fbfffd6486d7a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 552 is now running using a merge of 031f15c2aff68b9c6fbf0a25412fbfffd6486d7a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/552

**Comment by weld-tester** [comment]
Build 552 outcome was SUCCESS using a merge of 031f15c2aff68b9c6fbf0a25412fbfffd6486d7a on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/552
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/552

**Comment by jharting** [comment]
Merged, thanks!

**Pull request closed without merging**